packages/common/mui/src/lib/currency-input/currency-select.tsx

```diff
@@ -181,4 +195,37 @@ export const EmptyCurrencySelectOption = (theme: NotionalTheme) => (
   </StyledItem>
 );
 
+const StyledItem = styled(OptionUnstyled)(
```

Comment:
Does not seem like this matches the mock, there aren't any borders between items and the set height does not match what he wants, it should partially show the next currency on the initial pop up

![image](https://github.com/notional-finance/notional-monorepo/assets/977434/2ccb5707-ba2e-4208-89ec-6d4d3a258535)